docs/data-sources/cloud_backup_snapshot_export_job.md

```diff
@@ -49,7 +49,7 @@ In addition to all arguments above, the following attributes are exported:
 * `custom_data` - Custom data to include in the metadata file named `.complete` that Atlas uploads to the bucket when the export job finishes. Custom data can be specified as key and value pairs.
 * `components` - _Returned for sharded clusters only._ Export job details for each replica set in the sharded cluster.
 * `created_at` - Timestamp in ISO 8601 date and time format in UTC when the export job was created.
-* `err_msg` - Error message, only if the export job failed.
+* `err_msg` - Error message, only if the export job failed. **Note:** This attribute is deprecated as it is not being used.
```

Comment:
i just saw here: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2438
that we used a format like: **(DEPRECATED)** Use `replication_specs.#.region_config.#.

don't know which one is the best one, but we should unify